common/autoware_test_utils/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ ament_auto_add_library(autoware_test_utils SHARED
 ament_auto_add_library(mock_data_parser SHARED
   src/mock_data_parser.cpp)
 
+target_link_libraries(autoware_test_utils
+  yaml-cpp
+)
+
```

Comment:
This change is definitely needed as the build of this package consistently fails with a linking error.